### PR TITLE
Mongo & Elasticsearch: extend config templates with more options

### PR DESCRIPTION
**Pull request by gkze**
We need these options in production and are currently overriding the config. We'd like to have these available to configure.

### CODE REVIEW

File: templates/default/elastic.yaml.erb [hunk 1]
```diff
@@ -7,6 +7,9 @@ instances:
     <% unless i["pshard_stats"].nil? %>
     pshard_stats: <%= i["pshard_stats"] %>
     <% end %>
+    <% unless i["cluster_stats"].nil? %>
+      cluster_stats: <%= i["cluster_stats"] %>
```
Comment (olivielpeau) [inline templates/default/elastic.yaml.erb]: the indentation is not correct here: `cluster_stats` needs to have the same indentation as the other parameters. Could you correct it?

(same comment for the `ssl`  options of `mongo.yaml.erb`)

**Comment by gkze** [comment]
Good catch @olivielpeau, thank you. Corrected.

**Comment by olivielpeau** [comment]
For some reason I can't make the CI to re-run against a merge of this PR into the _current_ state of master, so the build still fails.

Merging this, I'll check that the resulting CI build on master is successful.

Thanks @gkze!

**Comment by gkze** [comment]
Thank you @olivielpeau 👍

**Pull request merged**